boa/environment.py

```diff
@@ -470,9 +470,22 @@ class FakeMessage(Message):  # Message object with settable attrs
         return self.vm.state.computation_class.apply_message(self.vm.state, msg, tx_ctx)
 
     # function to time travel
-    def time_travel(self, sleep_time):
-        self.vm.patch.timestamp += sleep_time
-        self.vm.patch.block_number += sleep_time // 12
+    def time_travel(
+        self,
+        seconds: Optional[int] = None,
+        blocks: Optional[int] = None,
+        block_delta: int = 12,
+    ):
+
+        if not (seconds or blocks):
+            raise ValueError("One of seconds or blocks should be set")
+        if seconds and not blocks:
+            blocks = seconds // block_delta
+        elif not seconds and blocks:
+            seconds = blocks * block_delta
```

Comment:
with the guard for `seconds is not None != blocks is not None`, these conditions can be simplified - to `if seconds is not None`, and `else`.

Reply:
```suggestion
        if seconds is not None:
            blocks = seconds // block_delta
        else:
            seconds = blocks * block_delta
```